crates/qt-build-utils/src/utils.rs

```diff
@@ -14,3 +16,16 @@ pub(crate) fn is_apple_target() -> bool {
 pub(crate) fn is_emscripten_target() -> bool {
     std::env::var("CARGO_CFG_TARGET_OS") == Ok("emscripten".to_owned())
 }
+
+/// Wrap a command in a native subshell
+pub(crate) fn native_shell_command(command: &str) -> Command {
```

Comment:
I believe passing the command as a plain string here will fail if the path to qmake contains a space.

As this path is provided by the user, we must ensure this works as well.
Please make sure that we can handle paths with spaces.

I suggest adding a testcase for this by adding a `test scripts/` folder that contains a `qmake.sh` and `qmake.bat` script that just echo a string or forward to `qmake`.